pkg/webhook/parser.go

```diff
@@ -493,6 +496,14 @@ func (g Generator) Generate(ctx *genall.GenerationContext) error {
 			if err != nil {
 				return err
 			}
+			for _, group := range cfg.Groups {
+				if group == "" { // aka "core"
+					continue
+				}
+				if !groupRegex.MatchString(group) {
```

Comment:
This error will be very frustrating for anyone that runs into it - Please include the string of the regex used for validation